### PR TITLE
Some updates to 2.1.50

### PR DESCRIPTION
**Pull request by cqg2**

### CODE REVIEW

File: src/browsing.md [hunk 1]
```diff
@@ -202,6 +202,7 @@ they can be found in a submenu of the context menu.
 | Name | Action |
 |-|-|
 | Add Notes | Open the [Add](editing.md#adding-cards-and-notes) dialog. |
+| Create Copy | Open a [duplicate](browsing.md#finding-duplicates) of the current note in the [editor](editing.md#adding-cards-and-notes), which can be slightly modified to easily obtain variations of your cards. By default, the duplicate card will be created in the same deck that the original. |
```
Comment (dae) [inline src/browsing.md:205]:
```suggestion
| Create Copy | Open a [duplicate](browsing.md#finding-duplicates) of the current note in the [editor](editing.md#adding-cards-and-notes), which can be slightly modified to easily obtain variations of your cards. By default, the duplicate card will be created in the same deck as the original. |
```

File: src/browsing.md [hunk 2]
```diff
@@ -224,7 +225,7 @@ they can be found in a submenu of the context menu.
 |-|-|
 | Change Deck | Move currently selected cards to a different deck. |
 | Set Due Date | Turn cards into review cards, and make them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like `60-90` will make the selected cards due between 60 and 90 days from now. New cards will have their interval set to the same delay, but reviews will be rescheduled without changing their current interval, unless '!' is included at the end of the range.
-| Forget | Move currently selected cards to the end of the new queue. The existing review history is preserved. |
+| Forget | Move currently selected cards to the end of the new queue. The existing review history is preserved. In 2.1.50+, you have the options to restore the original card position, and to reset the card's lapse and repetition counters.|
```
Comment (dae) [inline src/browsing.md:228]:
```suggestion
| Forget | Move currently selected cards to the end of the new queue. The existing review history is preserved. In 2.1.50+, there are options to restore the original card position, and to reset the card's lapse and repetition counters.|
```

File: src/deck-options.md [hunk 3]
```diff
@@ -17,7 +17,13 @@ Deck options are accessed by:
 
 This page describes the options shown in Anki 2.1.45+, when you have the v2 or
 v3 scheduler enabled. On older versions, some options will not be available, or
-will appear in a different section.
+will appear in a different section. Please, keep in mind that the V1 scheduler
```
Comment (dae) [inline src/deck-options.md:20]:
```suggestion
will appear in a different section. Please keep in mind that the V1 scheduler 
```

File: src/deck-options.md [hunk 4]
```diff
@@ -17,7 +17,13 @@ Deck options are accessed by:
 
 This page describes the options shown in Anki 2.1.45+, when you have the v2 or
 v3 scheduler enabled. On older versions, some options will not be available, or
-will appear in a different section.
+will appear in a different section. Please, keep in mind that the V1 scheduler 
+is no longer supported in Anki 2.1.50+. If you have not yet updated to V2 or V3,
+you will be prompted to update when you attempt to review cards in 2.1.50+
```
Comment (dae) [inline src/deck-options.md:22]:
```suggestion
you will be prompted to update when you attempt to review cards in 2.1.50+.
```

File: src/deck-options.md [hunk 5]
```diff
@@ -88,29 +94,17 @@ Controls how many new cards are introduced each day you open the program. If you
...
+- In the v1 and v2 schedulers, if you
+have decks in a parent-child-grandchild arrangement, both the parent and
+child limits will alter how many cards are shown from the grandchild, **even if you
+click directly on the grandchild.**
+
+- In the v3 scheduler, intermediate deck limits
+also affect their children, but if you click directly on the grandchild deck, 
+child and parent limits will be ignored. Please see the 
+[the v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits)
+page for more info.
```
Comment (dae) [inline src/deck-options.md:107]: These two paragraphs feel a bit confusing, and I'm not sure the comment about v2 is entirely correct. How does the following sound?

When decks are nested (eg `parent`, `parent::child`, `parent::child::grandchild`), the way the limits are applied depends on the scheduler version.

- V1 applies parent limits to children, regardless of which deck you click on
- V2 behaves similarly to V1 for new cards. For reviews, only the limits of the deck you click on are honored.
- V3 honors the limits of the deck you click on, and any decks inside it. Limits from parents above the deck you clicked on are ignored.

For more information, please see the 
[the v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits) page.

File: src/preferences.md [hunk 6]
```diff
@@ -7,13 +7,26 @@ the Anki menu on a Mac.
 
 ## Basic
 
+**Language**  
+Change your display language. You can help to iomprove translations [here](https://translating.ankiweb.net/)
```
Comment (dae) [inline src/preferences.md:11]:
```suggestion
Change your display language. You can help to improve translations [here](https://translating.ankiweb.net/)
```

File: src/preferences.md [hunk 7]
```diff
@@ -7,13 +7,26 @@ the Anki menu on a Mac.
...
+
+**Theme**  
+Dark (night) mode will cause Anki to show cards as white text on a
```
Comment (dae) [inline src/preferences.md:14]:
```suggestion
Dark (night) mode will make Anki's interface dark, and will cause cards to be shown as white text on a
```

File: src/preferences.md [hunk 8]
```diff
@@ -32,11 +45,7 @@ file sizes.
 By default, formatting like bold and colors are kept when pasting,
 unless the <kbd>Shift</kbd> key is held down. This option reverses the behaviour.
 
-**Night mode**  
-Night mode will cause Anki to show cards as white text on a
-black background. Some card templates may need to be modified to work
-properly with this option enabled - please see [night mode
-styling](templates/styling.md#night-mode) for more information.
+**Ignore accents in search**
```
Comment (dae) [inline src/preferences.md:48]:
```suggestion
**Ignore accents in search**

Simple text searches automatically [ignore accents](./searching.md#ignoring-accentscombining-characters).
```

File: src/preferences.md [hunk 9]
```diff
@@ -113,3 +122,7 @@ The network tab contains options related to syncing with AnkiWeb.
   ask you whether you wish to upload or download. This is useful if
   you have made some changes accidentally, and wish to overwrite them
   with an older version that is on AnkiWeb.
+
+## Backups
+
+Please, see [this](backups.md#automatic-backups) section of the manual
```
Comment (dae) [inline src/preferences.md:128]:
```suggestion
Please see [this](backups.md#automatic-backups) section of the manual.
```

File: src/studying.md [hunk 10]
```diff
@@ -198,6 +198,8 @@ cards of the same note.
 
    With the [2.1 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
    however, burying cards does not reset a card's learning steps.
+   
+- **Forget card**: Move current card to [the end of the new queue.](browsing.md#cards)
```
Comment (dae) [inline src/studying.md:202]:
```suggestion
- **Forget card**: Move current card to [the end of the new queue](browsing.md#cards).
```

File: src/studying.md [hunk 11]
```diff
@@ -216,11 +218,17 @@ cards of the same note.
...
+- **Previous Card Info**: Displays [statistical information](stats.md#card-info) about the previous card.
+
 - [**Mark Note**](editing.md#the-marked-tag): Adds a “marked” tag to the current note, so it can be easily found in the
 browser. This is similar to flagging individual cards, but works with a tag
 instead, so if the note has multiple cards, all cards will appear in a search
 for the marked tag. Most users will want to use flags instead. 
 
+- **Create Copy**: Opens a [duplicate](browsing.md#finding-duplicates) of the current 
+note in the editor, which can be slightly modified to easily obtain variations of your cards. 
+By default, the duplicate card will be created in the same deck that the original.
```
Comment (dae) [inline src/studying.md:230]:
```suggestion
By default, the duplicate card will be created in the same deck as the original. 
```

**Review by dae — COMMENTED** [review]
Thanks cqg! Some comments below.

File: src/deck-options.md (hunk 5 above)
Comment (cqg2) [reply src/deck-options.md]: Done (I wasn't sure about the v2, thanks for clarifying!)

File: src/preferences.md (hunk 6 above)
Comment (cqg2) [reply src/preferences.md:11]: Sorry for the mistakes (and thanks for the corrections) :)

File: src/preferences.md [hunk 12]
```diff
@@ -7,13 +7,26 @@ the Anki menu on a Mac.
...
+From 2.1.50+ there is an option to switch to day or night mode automatically. 
+
 **Video driver**  
 Anki's libraries need a video driver to draw content on the screen.
 Due to different hardware and software configurations, the driver that
 works best on your machine may vary. ANGLE and OpenGL will perform better
 than the software option, but they may not function correctly on some
 systems. On Macs, you will almost always want to use the OpenGL option.
 
+Note: If you're using 2.1.50+ (Qt6), please, check [this](./platform/windows/display-issues.md#qt6)
```
Comment (dae) [inline src/preferences.md:28]:
```suggestion
Note: If you're using 2.1.50+ (Qt6), please, check [this page](./platform/windows/display-issues.md#qt6).
```

File: src/preferences.md [hunk 13]
```diff
@@ -32,11 +45,9 @@ file sizes.
...
 unless the <kbd>Shift</kbd> key is held down. This option reverses the behaviour.
 
-**Night mode**  
-Night mode will cause Anki to show cards as white text on a
-black background. Some card templates may need to be modified to work
-properly with this option enabled - please see [night mode
-styling](templates/styling.md#night-mode) for more information.
+**Ignore accents in search**
+
+Simple text searches automatically [ignore accents](./searching.md#ignoring-accentscombining-characters).
```
Comment (dae) [inline src/preferences.md:50]:
```suggestion
When enabled, simple text searches automatically [ignore accents](./searching.md#ignoring-accentscombining-characters).
```

**Review by dae — COMMENTED** [review]
Thanks for adding these cqg! I'll make a couple of minor tweaks and then merge this in.

**Pull request merged**